src/commands/key-load-server-key.ts

```diff
@@ -0,0 +1,43 @@
+import * as Vorpal from 'vorpal'
+
+import * as utils from '../index'
+
+import Command from './Command'
+
+/**
+ * Loads a server private key and public certificate from files (in PEM format).
+ * The loaded keys are placed in local storage for use by the sign command.
+ */
+export default class LoadServerKeyCommand extends Command {
+  /**
+   * @override
+   */
+  protected async action(args: Vorpal.Args): Promise<void> {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment -- Vorpal.Args isn't typed
+    const keyPath: string = args.keyPath
+    const key = await utils.getSigningKeyFromFile(keyPath)
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment -- Vorpal.Args isn't typed
+    const certPath: string = args.certPath
+    this.vorpal.log(`loading server-key from ${keyPath}`)
+    this.vorpal.log(`loading server-cert from ${certPath}`)
+    const cert = await utils.getJwkFromFile(certPath)
+
+    this.localStorage.setSigningKey('server-key', key.toJWK(true))
+    this.localStorage.setSigningKey('server-cert', cert)
```

Comment:
So the key is stored in PEM format, but the cert is stored in JWK format?

Reply:
The cert is already a JWK by virtual of the call on line 23. The server key however is a RSAKey and needs to be converted to a JWK.